agent/app/agent_capability.go

```diff
@@ -245,6 +249,16 @@ func (agent *ecsAgent) appendTaskENICapabilities(capabilities []*ecs.Attribute)
 			})
 		}
 	}
+
+	if agent.cfg.ENITrunkingEnabled {
+		capabilities = appendNameOnlyAttribute(capabilities, attributePrefix+taskENITrunkingAttributeSuffix)
+
+		branchENIVersionAttribute, err := agent.getBranchENIPluginVersionAttribute()
+		if err != nil {
```

Comment:
I'd prefer to refactor append ENI trunking into a separate method.

I think if someone added a new capability X below this. And ENI trunking somehow got error. It will never execute capability X blog at all because of this error check.

Reply:
ok. updated